CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.20.0 (June 16, 2025)
+
+NOTES:
+
+* **This release requires Juju controller version 2.9.49 or higher Juju.**
+* **If using JAAS, this release requires Juju controller version 3.6.5 or higher.**
```

Comment:
How come 3.6.5? Wouldn't say 3.6.1 work?

Reply:
i thought we fixed some things around offers in 3.6.5